src/test/scala/io/dockstore/DockstoreWebUser.scala

```diff
@@ -60,8 +70,31 @@ class DockstoreWebUser extends Simulation {
       SearchPage.search,
       TRS.searchAndDrillDown,
       Metadata.go
-    )
+    ),
+
+    scenario(loggedInUserFlow_1_9).feed(tokenFeeder).exec(
+      LoggedInHomepage.loggedInHomepage,
+      Accounts.accountsPage,
+      io.dockstore.release_1_9.SearchPage.search,
+      Organizations.organizations,
+      CreateAndUpdateHostedWorkflow.create,
+      HostedWorkflows.fetchRandomAndTogglePublish,
+      MyWorkflows.myWorkflows,
+      StarredToolsAndWorkflows.page
+    ),
+
+    scenario(loggedOutUserFlow_1_9).exec(
+      LoggedOutHomepage.loggedOutHomepage,
+      io.dockstore.release_1_9.SearchPage.search,
+      TRS.searchAndDrillDown,
+      Metadata.go
+    ),
 
+    scenario("jamboree").feed(tokenFeeder).exec(
+      LoggedInHomepage.loggedInHomepage,
```

Comment:
Are most people going to be logged in at the Jamboree? Isn't the Jamboree searching for a workflow rather than creating your own?